doc/userguide/upgrade.rst

```diff
@@ -37,6 +37,12 @@ Major changes
 ~~~~~~~~~~~~~
```

Comment:
would expect doc about this topic also outside of an 'upgrade' doc

Reply:
@jufajardini should we have a security.rst in the user guide ? 
Or a Config hardening ?

Reply:
Yes :)

Reply:
(TIL config hardening :o )
I think I prefer the idea of a config hardening chapter/section

Reply:
Proposing something in next PR